gp-includes/formats/format-php.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * GlotPress Format PHP class
+ *
+ * @since 4.0.0
+ *
+ * @package GlotPress
+ */
+
+/**
+ * Format class used to support PHP file format.
+ *
+ * @since 4.0.0
+ */
+class GP_Format_PHP extends GP_Format {
+	/**
+	 * Name of file format, used in file format dropdowns.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @var string
+	 */
+	public $name = 'PHP (.php)';
+
+	/**
+	 * File extension of the file format, used to autodetect formats and when creating the output file names.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @var string
+	 */
+	public $extension = 'php';
+
+	/**
+	 * Generates a string the contains the $entries to export in the PHP file format.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @param GP_Project         $project         The project the strings are being exported for, not used
+	 *                                            in this format but part of the scaffold of the parent object.
+	 * @param GP_Locale          $locale          The locale object the strings are being exported for. not used
+	 *                                            in this format but part of the scaffold of the parent object.
+	 * @param GP_Translation_Set $translation_set The locale object the strings are being
+	 *                                            exported for. not used in this format but part
+	 *                                            of the scaffold of the parent object.
+	 * @param GP_Translation     $entries         The entries to export.
+	 * @return string The exported PHP string.
+	 */
+	public function print_exported_file( $project, $locale, $translation_set, $entries ) {
+		$language_code = $this->get_language_code( $locale );
+		if ( false === $language_code ) {
+			$language_code = $locale->slug;
+		}
+
+		$current        = $project;
+		$project_tree   = array();
+		$project_tree[] = $current->name;
+
+		while ( $current->parent_project_id > 0 ) {
+			$current        = GP::$project->get( $current->parent_project_id );
+			$project_tree[] = $current->name;
+		}
+
+		$project_tree = array_reverse( $project_tree );
+
+		$project_id_version = implode( ' - ', $project_tree );
+
+		/** This filter is documented in gp-includes/formats/format-pomo.php */
+		$project_id_version = apply_filters( 'gp_pomo_export_project_id_version', $project_id_version, $project_tree );
```

Comment:
Related with this filter, it was thought to be used in the `pomo`, not for the `php` export.

What do you think about creating a new filter, only for the PHP export?

```suggestion
		/**
		 * Filter the project name and version header before export.
		 *
		 * @since 4.0.0
		 *
		 * @param string $project_id_version The default project name/version to use in the header and
		 *                                   comments ( "Parent - Child - GrandChild - etc." by default).
		 * @param array  $project_tree       An array of the parent/child project tree, ordered from Parent
		 *                                   to child to grandchild to etc...
		 */
		$project_id_version = apply_filters( 'gp_php_export_project_id_version', $project_id_version, $project_tree );
```